### DISCUSSION TITLE
Is this repository abandoned or just paused without notice?

### DISCUSSION THREAD

**Discussion by mashb1t** [Stability-AI/StableStudio#117] [General]
- The last commit was on June 1 (https://github.com/Stability-AI/StableStudio/commit/76772d7b9a4e6d87bd87c26665f7ab74cd40f2cd)
- There seems to be no real support for SDXL with A1111 API as basically no 1024x1024 image support is in the code by default.
- Also the editor does only support 512x512 canvas renderings when adding new elements.
- Discord link is dead (https://discord.com/channels/1002292111942635562/1108055793674227782)

Do you have any information about the future of this repository?

related to https://github.com/Stability-AI/StableStudio/discussions/106#discussioncomment-6646832

**Comment by ser** [comment]
it doesn't look good, does it.
